iina/uk.lproj/PrefUIViewController.strings

```diff
@@ -155,7 +155,7 @@ Picture:"; ObjectID = "G8w-J7-1q1"; */
 "bYj-5g-zYO.title" = "Клітинка";
 
 /* Class = "NSButtonCell"; title = "Toggle Picture-in-Picture by minimizing/un-minimizing the window"; ObjectID = "baY-O6-fcB"; */
-"baY-O6-fcB.title" = "Перемкнути «Кадр у кадрі» згортанням / розгортанням вікна";
+"baY-O6-fcB.title" = "Перемкнути режим «Картинка в картинці» згортанням / розгортанням вікна";
```

Comment:
Wrong. The red one is used in official Ukrainian localisation of macOS apps (eg., Safari web browser).
<img width="292" alt="Screenshot 2019-05-06 16 38 04" src="https://user-images.githubusercontent.com/36358174/57229233-3ca09f00-701e-11e9-9104-12819044c9e9.png">